image/templates/helm/stackrox-central/templates/_central_setup.tpl

```diff
@@ -52,26 +52,35 @@
     Central's DB PVC config setup
   */}}
 {{ $dbVolumeCfg := dict }}
+{{ $dbBackupVolumeCfg := dict }}
 {{ if not $centralDBCfg.external }}
 {{ if $centralDBCfg.persistence.none }}
   {{ include "srox.warn" (list $ "You have selected no persistence backend. Every deletion of the StackRox Central DB pod will cause you to lose all your data. This is STRONGLY recommended against.") }}
   {{ $_ := set $dbVolumeCfg "emptyDir" dict }}
+  {{ $_ := set $dbBackupVolumeCfg "emptyDir" dict }}
 {{ end }}
 {{ if $centralDBCfg.persistence.hostPath }}
   {{ if not $centralDBCfg.nodeSelector }}
     {{ include "srox.warn" (list $ "You have selected host path persistence, but not specified a node selector. This is unlikely to work reliably.") }}
   {{ end }}
   {{ $_ := set $dbVolumeCfg "hostPath" (dict "path" $centralDBCfg.persistence.hostPath) }}
+  {{ $_ := set $dbBackupVolumeCfg "hostPath" (printf "%s/backup" (dict "path" $centralDBCfg.persistence.hostPath)) }}
 {{ end }}
 {{/* Configure PVC if either any of the settings in `centralDB.persistence.persistentVolumeClaim` are provided,
      or no other persistence backend has been configured yet. */}}
 {{ if or (not (deepEqual $._rox._configShape.central.db.persistence.persistentVolumeClaim $centralDBCfg.persistence.persistentVolumeClaim)) (not $dbVolumeCfg) }}
   {{ $dbPVCCfg := $centralDBCfg.persistence.persistentVolumeClaim }}
+  {{ $dbBackupPVCCfg := $centralDBCfg.persistence.persistentVolumeClaim }}
   {{ $_ := include "srox.mergeInto" (list $dbPVCCfg $._rox._defaults.dbPVCDefaults (dict "createClaim" (or .Release.IsInstall (eq $._rox._renderMode "centralDBOnly")))) }}
+  {{ $_ := include "srox.mergeInto" (list $dbBackupPVCCfg $._rox._defaults.dbPVCDefaults (dict "createClaim" (or .Release.IsInstall (eq $._rox._renderMode "centralDBOnly") $centralDBCfg.persistence.createBackup ))) }}
   {{ $_ = set $dbVolumeCfg "persistentVolumeClaim" (dict "claimName" $dbPVCCfg.claimName) }}
+  {{ $_ = set $dbBackupVolumeCfg "persistentVolumeClaim" (dict "claimName" (printf "%s-backup" $dbPVCCfg.claimName)) }}
   {{ if $dbPVCCfg.createClaim }}
     {{ $_ = set $centralDBCfg.persistence "_pvcCfg" $dbPVCCfg }}
   {{ end }}
+  {{ if $dbBackupPVCCfg.createClaim }}
+    {{ $_ = set $centralDBCfg.persistence "_backupPVCCfg" $dbBackupPVCCfg }}
+  {{ end }}
```

Comment:
This seems unused. Drop?

Reply:
Why do you think it's unused?

Reply:
Because I could not find any ocurrence of `_backupPVCCfg` elsewhere.

Reply:
Ah, right. Those are leftovers from my attempt to add the backup PVC directly into the Helm chart.